src/Searcher.tsx

```diff
@@ -6,18 +6,11 @@ import { LibraryContainer } from "./components/LibraryContainer";
 import * as LibraryUtilities from "./LibraryUtilities";
 import { SearchBar } from "./components/SearchBar";
 
-type displayMode = "structure" | "list";
-
-interface SearchModeChangedFunc {
-    (inSearchMode: boolean): void;
-}
-
```

Comment:
The PR description does not tell why this is removed, can you add that reason to the description? Can't seem to recall why is this done...